### PR TITLE
IA-3625 Mobile setup zip: Add missing API calls

### PR DESCRIPTION
**Pull request by bramj**
Added files for the groupsets and deleted entities API calls.
And a (very) small improvement to the task's progress message.

Related JIRA tickets : https://bluesquare.atlassian.net/browse/IA-3625

### CODE REVIEW

File: iaso/tasks/utils/mobile_app_setup_api_calls.py [hunk 1]
```diff
@@ -7,6 +7,10 @@
         "path": "/api/mobile/groups/",
         "filename": "groups",
     },
+    {
+        "path": "/api/mobile/group_sets/",
```
Comment (bmonjoie) [inline iaso/tasks/utils/mobile_app_setup_api_calls.py]: It's missing: `?showDeleted=true`

File: iaso/tasks/utils/mobile_app_setup_api_calls.py [hunk 2]
```diff
@@ -7,6 +7,10 @@
         "path": "/api/mobile/groups/",
```
Comment (bmonjoie) [inline iaso/tasks/utils/mobile_app_setup_api_calls.py]: It's missing: `?showDeleted=true`

**Review by bmonjoie — CHANGES_REQUESTED** [review]

**Review by kemar — APPROVED** [review]
LGTM but no clue about `?showDeleted=true`?

**Comment by bmonjoie** [comment]
@kemar  The mobile must be aware of the deleted groups and groupSets because otherwise, it could receive old change requests with unknown groups/groupSets.

**Review by bmonjoie — APPROVED** [review]

**Comment by bramj** [comment]
Just became a hotfix because urgently needed for a demo

**Pull request merged**